src/components/pages/Register/Register.module.css

```diff
@@ -15,9 +16,9 @@
 }
 
 .input {
-  width: 33vw;
+  width: 27vw;
   max-width: 600px;
-  height: clamp(30px, 5vh, 50px);
```

Comment:
This totally isnt a high priority thing but we should use clamp so that on really big screens the component has some kind of hard limit on how big / small it can get it.
You can read about the clamp function which does just that [here](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp). PS dont feel about not knowing about it if you don't, I just learned about it a couple weeks ago.